README.md

```diff
@@ -109,20 +109,6 @@ If you want to add a new npm package to 'node_modules':
     }
     ```
 
-## Module examples
-
-Check out these module types that you can use as a blueprint.
-
-* [Table](https://github.com/dcos/dcos-ui/blob/master/src/js/components/ServicesTable.js)
```

Comment:
we could but then we're in the middle of modularization and new links won't give a comprehensive picture. If you still think it's better to keep them - I can do so.